spec/rubocop/cop/style/access_modifier_declarations_spec.rb

```diff
@@ -735,6 +735,35 @@ class Test
             end
           RUBY
         end
+
+        it 'does not delete methods before the group declaration' do
```

Comment:
Can you update it? The description in `it` should only state the condition under which an offense is registered.

```suggestion
        it 'registers an offense when using a grouped access modifier declaration' do
```

Reply:
That's a great improvement, I couldn't figure out what to describe this with - thanks for the suggestion, done 👍🏻